File: BaekJoon/ITBigTech/CookieMeasure/CookieMeasure.py
def main():
    N = int(input())

    rows = [input().strip() for _ in range(N)]

    head = []

    # head 찾기
    # 이거 N말고 2로 했더니 런타임에러(IndexError) 났음
    for i in range(N):
        if "*" in rows[i]:
            head.append(i+2)
            head.append(rows[i].index("*")+1)
            break

    left_hand = 0
    right_hand = 0
    waist = 0
    left_leg = 0
    right_leg = 0

    for i in range(head[0]-1, N):
        # 팔 길이
        if i == head[0]-1:
            for j in range(head[1]-1):
                if rows[i][j] == "*":
                    left_hand += 1
            for j in range(head[1], len(rows[0])):
                if rows[i][j] == "*":
                    right_hand += 1
        # 허리 길이
        elif rows[i][head[1]-1] == "*":
            waist += 1
        # 다리 길이
        else:
            if head[1] > 1 and rows[i][head[1] - 2] == "*":
                left_leg += 1
            if head[1] < len(rows[0]) and rows[i][head[1]] == "*":
                right_leg += 1
        


    print(head[0],head[1])
    print(left_hand, right_hand, waist, left_leg, right_leg)

if __name__ == "__main__":
    main()
                
        

        

File: BaekJoon/ITBigTech/CookieMeasure/CookieMeasure.java
import java.util.*;

public class CookieMeasure {
    public static void main(String args[]){
	    Scanner sc = new Scanner(System.in);	
        int N = Integer.parseInt(sc.nextLine().trim());

        ArrayList<String> rows = new ArrayList<>();

        // 입력 저장
        for (int i = 0; i < N; i++) {
            rows.add(sc.nextLine());
        }

        // 머리 저장
        int[] head = new int[2];
        for (int i = 0; i < N; i++) {
            if (rows.get(i).contains("*")) {
                head[0] = i+2;
                head[1] = rows.get(i).indexOf("*") + 1;
                break;
            }
        }

        // 몸
        int leftHand = 0, rightHand = 0, waist = 0, leftLeg = 0, rightLeg = 0;
        
        for (int i = head[0] - 1; i < N; i++) {

            // 팔
            if (i == head[0] - 1) {
                // 왼팔
                for (int j = 0; j < head[1]-1; j++) {
                    if (rows.get(i).charAt(j) == '*') {
                        leftHand++;
                    }
                }
                // 오른팔
                for (int j = head[1]; j < rows.get(i).length(); j++) {
                    if (rows.get(i).charAt(j) == '*') {
                        rightHand++;
                    }
                }
            }

            if (i >= head[0]) {
                // 허리
                if (rows.get(i).charAt(head[1]-1) == '*') waist++;

                // 다리
                if (rows.get(i).charAt(head[1]-2) == '*' && rows.get(i).charAt(head[1]) == '*') {
                    leftLeg++;
                    rightLeg++;
                } else if (rows.get(i).charAt(head[1]-2) == '*') leftLeg++;
                else if (rows.get(i).charAt(head[1]) == '*') rightLeg++;
            }
        }

        System.out.printf("%s %s\n", head[0], head[1]);
        System.out.printf("%d %d %d %d %d\n", leftHand, rightHand, waist, leftLeg, rightLeg);
        sc.close();
    }
}

